### DISCUSSION TITLE
Checkbox on selects

### DISCUSSION THREAD

**Discussion by MaximoManzotti** [JedWatson/react-select#5752] [Ideas]
the library should have a option to put checkbox on the dropdown list options. if you have a multi select when you open you can see the checked options

**Comment by manjushsh** [comment]
HI @MaximoManzotti , you can already implement same thing using custom components.

**Reply by MaximoManzotti** [reply]
Yes i know but it's work that you can use in other thing, if they implemented this i think that will help us
